website/content/docs/upgrading/deduplication/different-case.mdx

```diff
@@ -0,0 +1,184 @@
+---
+layout: docs
+page_title: Resolve different-case entity alias duplicates
+description: >-
+  Fix duplicate identities for entity aliases due to case differences.
+---
+
+# Fix different-case entity alias duplicates
+
+Fix duplicate identities for entity aliases due to case differences.
+
+**You must review different-case entity alias duplicates before enabling forced
+identity deduplication due to the potential security risk of incorrectly merging
+distinct aliases**.
+
+
+<Tip title="Assumptions">
+
+- You have [identified the duplicate entity aliases](/vault/docs/upgrading/identity-deduplication). 
+- You have admin permission on the relevant Vault server or cluster.
+
+</Tip>
+
+
+## Why entity alias duplicates happen
+
+Historical bugs in Vault allowed authN mounts to create duplicate entries of the
+same entity alias if the aliases used difference cases. For example, if an
+external LDAP system returns unnormalized usernames or changes the normalization
+scheme over time. By default, Vault uses case-insensitive matching for the
+usernames and considers `username` and `uSeRnAmE` to be the same alias. But,
+past bugs have left some users with different-case duplicates in storage. To
+avoid the security impact of incorrectly merging identities that do not
+represent the same user, Vault tries to honor past behavior by switching the
+identity engine to use case-sensitive matching mode.
+But case-sensitive matching 
+This mode is not well supported however  
+as it deviates from Vault's typical assumptions. Remaining in this mode can lead to unexpected behavior. 
```

Comment:
I think this summary which was part of my above suggestion ended up broken in the edit somewhere. I think the wording is good once the partial sentence is removed and this is merged with the final sentence of the paragraph above.